lib/spark_api/models/listing.rb

```diff
@@ -235,6 +235,10 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
+      def batch_photo_delete(photoIds, args={})
```

Comment:
I think we are of a similar mind here, @wlmcewen. This feels like it's in a grey-ish area.

The reorder methods I mentioned earlier follow the ActiveRecord idioms and the idioms @ryanhertz pointed out as well. I think we'd be well-served to follow the pattern of those methods and use `delete_photos` (wraps `delete_photos!`, handles known errors, logs, etc) and `delete_photos!` (rename this implementation of `batch_photo_delete`).

That approach would give us consistency in naming and implementation with photo stuff in this class.